src/sentry/web/frontend/setup_wizard.py

```diff
@@ -132,3 +107,45 @@ def get(self, request: HttpRequest, wizard_hash) -> HttpResponse:
 
         context["organizations"] = serialize(list(orgs))
         return render_to_response("sentry/setup-wizard.html", context, request)
+
+
+def get_token(orgs: [Organization], user: User):
```

Comment:
```suggestion
def get_token(orgs: list[Organization], user: User):
```

Is there a container type missing here?